.travis.yml

```diff
@@ -15,7 +15,7 @@ script:
   - scripts/update_docs.sh
 
 after_success:
-  - tox -e coveralls
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
That's how they say to do it.
![screen shot 2016-06-20 at 4 32 31 pm](https://cloud.githubusercontent.com/assets/192456/16209352/a064b054-3704-11e6-8722-f88b6f3dc11d.png)